File: 0226-invert-binary-tree/0226-invert-binary-tree.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right
class Solution:
    def invertTree(self, root: Optional[TreeNode]) -> Optional[TreeNode]:
        if root == None:
            return root
        l = root.left
        r = root.right
        
        root.left = r
        root.right = l
        self.invertTree(root.left)
        self.invertTree(root.right)    
        return root

File: 0226-invert-binary-tree/0226-invert-binary-tree.js
/**
 * Definition for a binary tree node.
 * function TreeNode(val, left, right) {
 *     this.val = (val===undefined ? 0 : val)
 *     this.left = (left===undefined ? null : left)
 *     this.right = (right===undefined ? null : right)
 * }
 */
/**
 * @param {TreeNode} root
 * @return {TreeNode}
 */

var invert = function(root){
    if(root == null) return;
    invert(root.left)
    invert(root.right)
    
    let temp = root.left
    root.left = root.right
    root.right = temp
}

var invertTree = function(root) {
    invert(root);
    return root
};